sdk/resourcemanager/azure-resourcemanager-privatedns/pom.xml

```diff
@@ -0,0 +1,134 @@
+<!--
+ Copyright (c) Microsoft Corporation. All rights reserved.
+ Licensed under the MIT License. See License.txt in the project root for
+ license information.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-client-sdk-parent</artifactId>
+      <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+      <relativePath>../../parents/azure-client-sdk-parent</relativePath>
```

Comment:
It should do format. 4 indent instead

Reply:
does the repo has formatting setting such as in vscode or intellij checked in?

Reply:
No, all are formatted by ourselves